docs/references/database/delete.md

```diff
@@ -0,0 +1 @@
+Delete a database by its unique ID. Only users with write permissions have access to delete a database.
```

Comment:
In database we don't have permission, only API key with `databases.write` scope can delete the database

Reply:
We should use the same description as we use for other similar resources. A Function might be a good candidate. `Delete a function by its unique ID.`

Reply:
>We should use the same description as we use for other similar resources. A Function might be a good candidate. Delete a function by its unique ID.

This will be a separate task to look to standardize description

> In database we don't have permission, only API key with databases.write scope can delete the database

Addressed